cmd/pyroscope/command/usage.go

```diff
@@ -32,44 +32,45 @@ var hiddenCommands = []string{
 	"dbmanager",
 }
 
+// TODO: Do we want to keep this or use cobra default one? Maybe banner + cobra default? Or something else?
 // This is mostly copied from ffcli package
-func DefaultUsageFunc(sf *SortedFlags, c *ffcli.Command, deprecatedFields []string) string {
+func DefaultUsageFunc(sf *pflag.FlagSet, c *cobra.Command) string {
 	var b strings.Builder
 
 	fmt.Fprintf(&b, "continuous profiling platform\n\n")
 	headerClr.Fprintf(&b, "USAGE\n")
-	if c.ShortUsage != "" {
-		fmt.Fprintf(&b, "  %s\n", c.ShortUsage)
+	if c.Use != "" {
+		fmt.Fprintf(&b, "  %s\n", c.Use)
 	} else {
-		fmt.Fprintf(&b, "  %s\n", c.Name)
+		fmt.Fprintf(&b, "  %s\n", c.Name())
 	}
 	fmt.Fprintf(&b, "\n")
 
-	if c.LongHelp != "" {
-		fmt.Fprintf(&b, "%s\n\n", c.LongHelp)
+	if c.Long != "" {
+		fmt.Fprintf(&b, "%s\n\n", c.Long)
 	}
 
-	if len(c.Subcommands) > 0 {
+	if c.HasSubCommands() {
 		headerClr.Fprintf(&b, "SUBCOMMANDS\n")
 		tw := tabwriter.NewWriter(&b, 0, 2, 2, ' ', 0)
-		for _, subcommand := range c.Subcommands {
-			if !slices.StringContains(hiddenCommands, subcommand.Name) {
-				fmt.Fprintf(tw, "  %s\t%s\n", itemClr.Sprintf(subcommand.Name), subcommand.ShortHelp)
+		for _, subcommand := range c.Commands() {
+			if !slices.StringContains(hiddenCommands, subcommand.Name()) {
```

Comment:
We should use `cobra.Command.Hidden` attribute. Instead of global `hiddenCommands` every such a command should be marked accordingly.

```suggestion
			if !subcommand.Hidden {
```